resources/openapi_v3.yaml

```diff
@@ -269,10 +321,7 @@ paths:
         content:
           application/json:
             schema:
-              anyOf:
-                - $ref: "#/components/schemas/AssignedMinistry"
-                - $ref: "#/components/schemas/Progress"
-                - $ref: "#/components/schemas/Field"
+              $ref: "#/components/schemas/Progress"
```

Comment:
Better have **status** instead of **progress**

Reply:
I will change those.